doc/nrf/protocols/matter/getting_started/low_power_configuration.rst

```diff
@@ -81,12 +81,29 @@ For devices with a short Slow Polling Interval, using this functionality can lea
 Short idle time and long idle time devices
 ==========================================
 
-The Matter v1.2 specification divides ICD into short idle time (SIT) and long idle time (LIT) devices.
+The Matter specification divides ICD into short idle time (SIT) and long idle time (LIT) devices.
 The division is based on the Slow Polling Interval parameter: equal to or shorter than 15 seconds for SIT, and longer than 15 seconds for LIT.
 
-The SIT device behavior is aligned with what was called SED in Matter v1.1 specification.
-The LIT device implementation requires multiple new features, such as Check-In protocol support and ICD client registration.
-The division of ICD types and related features were not finalized for Matter v1.2 and they are marked as provisional, so it is not recommended to use them, though you can find some of the LIT implementation in the Matter SDK and Matter specification.
+The typical use case for a SIT device are actuators, meaning devices such as door locks or window coverings, where the controller usually initiates communication and the accessory device is expected to answer with a small latency.
+Conversely, LIT devices are designed to be used for sensors or light switches, devices that only report data and are not controllable.
+In such scenarios, the LIT device initiates communication and it is not able to answer with a small latency, but it can sleep for long hours and achieve much lower power consumption than an SIT.
```

Comment:
```suggestion
In such scenarios, the LIT device initiates communication and it is not able to answer with a small latency, but it can sleep for extended periods of time and achieve much lower power consumption than an SIT.
```